streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -43,6 +41,8 @@
     private static final String TEST_ID = "reset-integration-test";
 
     static {
+        sslConfig = null;
```

Comment:
Why this change instead of setting `null` initially?

Reply:
The `sslConfig` is a static object of the underlying `AbstractResetIntegrationTest`, so in jenkins if `ResetIntegrationWithSSLTest` is triggered first `sslConfig` will be kept not null. This is from my observation in jenkins builds.